### ISSUE TITLE
Display "Loading..." text/animation for featured (homepage) topics

### ISSUE THREAD

**Issue by akshayinbox** [WikiDiggly/Diggly-Front-End#17]
Given parsing/gathering of homepage topics take quite some time, we could display the same animation or colored dots or a simple "Loading..." text while they load.

**Comment by brh55** [comment]
Yeah I'm considering adding a loading home screen before it shows the content
